beacon-chain/blockchain/process_block_helpers.go

```diff
@@ -118,7 +118,7 @@ func (s *Service) VerifyFinalizedBlkDescendant(ctx context.Context, root [32]byt
 			bytesutil.Trunc(root[:]), finalizedBlk.Slot(), bytesutil.Trunc(bFinalizedRoot),
 			bytesutil.Trunc(fRoot[:]))
 		tracing.AnnotateError(span, err)
-		return err
+		return invalidBlock{err}
```

Comment:
if the block is not part of the finalized descendent, I think it's safe to mark it as invalid, manually intervention will likely be involved at this point